cypress/support/constants/constants.js

```diff
@@ -199,6 +199,7 @@ module.exports = {
   LIFECYCLE_HISTORY_RESPONSE: 'Lifecycle history response fetched from application: ',
   LIFECYCLE_HISTORY_SCHEMA_PATH: 'schemas/lifecycleHistorySchema',
   LIFECYCLE_INTENT: 'Lifecycle intent sent to application: ',
+  LIFECYCLE_METHOD_LIST: ['Lifecycle.ready', 'Lifecycle.state', 'Lifecycle.close'],
   LIFECYCLE_NOTIFICATION_GENERATED: 'Lifecycle events generated Req #',
```

Comment:
Make use of the already present map https://github.com/rdkcentral/firebolt-certification-suite/pull/43/files#diff-9a74cc0669815673a532c7ffc61e3e438a1e169242089a49230abc39b0d2ef23R178

Reply:
The reason of keeping a seperate map for schema validation is because only these methods currently have schema in openrpc when checked. The rest of the methods do not have a schema and i dont think its possible to perform schema validation atm and will lead to failures. The idea was to add more methods to the list if their schema is available in future. Also lifecycle.finished does have schema additionally but there is below note in fca, so I wasnot sure whether lifecycle.finished should be added to the list atm. Let me know if my understanding is corrrect here
       ```
 /**
         * Directly calling finish is not an expected behavior of the app.
         * Finish should ideally be called by the app when unload event is generated.
         * For testing, we expect bolt to launch multiple apps and create a memory crunch
         * which would force the platform to generate and unload event.
         * TODO: Approach for the validation of lifecycle.finish() is yet to be decided.
         */

```